Makefile

```diff
@@ -122,42 +122,26 @@ e2e-helm-install:
 	./.staging/helm/linux-amd64/helm version --client
 
 e2e-helm-deploy: e2e-helm-install
-ifneq ($(GATEKEEPER_NAMESPACE),gatekeeper-system)
-	kubectl create namespace $(GATEKEEPER_NAMESPACE) --dry-run=client -o yaml | kubectl apply -f -
-	kubectl label ns $(GATEKEEPER_NAMESPACE) admission.gatekeeper.sh/ignore=no-self-managing
-	kubectl label ns $(GATEKEEPER_NAMESPACE) gatekeeper.sh/system="yes"
-	$(eval HELM_ARGS := --namespace $(GATEKEEPER_NAMESPACE) --set createNamespace=false)
-endif
-
-	@if [ $$(echo ${HELM_VERSION} | head -c 1) = "2" ]; then\
-		kubectl create clusterrolebinding tiller-admin --clusterrole=cluster-admin --serviceaccount=kube-system:default;\
-		./.staging/helm/linux-amd64/helm init --wait --history-max=5;\
-		kubectl -n kube-system wait --for=condition=Ready pod -l name=tiller --timeout=300s;\
-		./.staging/helm/linux-amd64/helm install manifest_staging/charts/gatekeeper --name=gatekeeper --debug ${HELM_ARGS} --set image.repository=${HELM_REPO} --set image.release=${HELM_RELEASE} --set emitAdmissionEvents=true --set emitAuditEvents=true;\
-	else\
-		./.staging/helm/linux-amd64/helm install manifest_staging/charts/gatekeeper --name-template=gatekeeper ${HELM_ARGS} --debug --set image.repository=${HELM_REPO} --set image.release=${HELM_RELEASE} --set emitAdmissionEvents=true --set emitAuditEvents=true;\
-	fi;
+	./.staging/helm/linux-amd64/helm install manifest_staging/charts/gatekeeper --name-template=gatekeeper -n ${GATEKEEPER_NAMESPACE} --create-namespace --debug \
+	--set image.repository=${HELM_REPO} \
+	--set image.release=${HELM_RELEASE} \
+	--set emitAdmissionEvents=true \
+	--set emitAuditEvents=true \
+	--set postInstall.labelNamespace.enabled=true;\
 
 e2e-helm-upgrade-init: e2e-helm-install
-	@if [ $$(echo ${HELM_VERSION} | head -c 1) = "2" ]; then\
-		kubectl create clusterrolebinding tiller-admin --clusterrole=cluster-admin --serviceaccount=kube-system:default;\
-		./.staging/helm/linux-amd64/helm init --wait --history-max=5;\
-		kubectl -n kube-system wait --for=condition=Ready pod -l name=tiller --timeout=300s;\
-		./.staging/helm/linux-amd64/helm repo add gatekeeper https://open-policy-agent.github.io/gatekeeper/charts;\
-		./.staging/helm/linux-amd64/helm install gatekeeper/gatekeeper \
-			--version ${BASE_RELEASE} --name gatekeeper \
-			--set emitAdmissionEvents=true --set emitAuditEvents=true --debug --wait;\
-	else\
 		./.staging/helm/linux-amd64/helm repo add gatekeeper https://open-policy-agent.github.io/gatekeeper/charts;\
-		./.staging/helm/linux-amd64/helm install gatekeeper/gatekeeper \
-			--version ${BASE_RELEASE} --name-template gatekeeper \
-			--set emitAdmissionEvents=true --set emitAuditEvents=true --debug --wait;\
-	fi;
+		./.staging/helm/linux-amd64/helm install gatekeeper gatekeeper/gatekeeper --version ${BASE_RELEASE} --debug --wait \
+		 --set emitAdmissionEvents=true \
+		 --set emitAuditEvents=true \
+		 --set customResourceDefinitions.create=false;\
```

Comment:
`customResourceDefinition.create` should be removed now?

Reply:
The reason why I left this was because this is the currently released chart being installed before we try and do an upgrade. So `customResourceDefinition.create` is still in that chart.